FZJDDFCH/FZJDDFCH-IOC-01App/Db/error_setter.substitutions

```diff
@@ -0,0 +1,34 @@
+file $(UTILITIES)/db/error_setter.template {
+    pattern {P, STREAM_PV, PV_NAME}
+    
+    {"\$(P)", "FREQ:REF", "FREQ:SP:RBV"}
+    {"\$(P)", "FREQ:REF", "FREQ"}
+    {"\$(P)", "FREQ:REF", "PHAS:SP:RBV"}
+    {"\$(P)", "FREQ:REF", "PHAS"}
+    {"\$(P)", "FREQ:REF", "PHAS:STATUS"}
+    {"\$(P)", "FREQ:REF", "MB"}
+    {"\$(P)", "FREQ:REF", "MB:STATUS"}
+    {"\$(P)", "FREQ:REF", "MB:INT"}
+    {"\$(P)", "FREQ:REF", "DRIVE"}
+    {"\$(P)", "FREQ:REF", "DRIVE:MODE"}
+    {"\$(P)", "FREQ:REF", "DRIVE:PHASE1:CURR"}
```

Comment:
Please be consistent with use of `PHAS` and `PHASE`

Reply:
`PHAS` refers to the chopper rotational phase, whereas `PHASE` refers to the motor windings.  However, I have renamed the PVs from `PHASE` to `L` (as per the manual) and edited their `.DESC` fields accordingly.